account.go

```diff
@@ -7,10 +7,10 @@ import (
 )
 
 const (
-	accountPath                        = "/auth/realms/:realm/account"
 	accountExtensionAPIPath            = "/auth/realms/master/api/account/realms/:realm"
-	accountPasswordPath                = accountExtensionAPIPath + "/credentials/password"
```

Comment:
why was this removed?

Reply:
No just moved 3 lines below... It was not logical to do accountExtensionAPIPath + "/credentials... rather than accountCredentialsPath + "...

Reply:
Not removed, just moved 3 lines below... It was not logical to do accountExtensionAPIPath + "/credentials... rather than accountCredentialsPath + "...

Reply:
Sorry, I did not see this